packages/api/db/migrations/20200608182453_create_users_table.ts

```diff
@@ -7,8 +7,9 @@ export const up = async (knex: Knex) => {
         table.string("email").notNullable();
         table.string("password").notNullable();
         table.string("role").notNullable();
-        table.integer("rank").notNullable();
+        table.integer("exp").notNullable();
         table.string("description");
+        table.specificType("tutorials", "INT[]").notNullable();
     });
```

Comment:
Oof you should've created another migrations file but whatev we dont have any data in the db for now

Reply:
Yeah, I created it before BG told us about the separate migrations.

Reply:
Ok its alright